gnovm/pkg/gnolang/alloc.go

```diff
@@ -299,3 +333,80 @@ func (alloc *Allocator) NewHeapItem(tv TypedValue) *HeapItemValue {
 	alloc.AllocateHeapItem()
 	return &HeapItemValue{Value: tv}
 }
+
+// -----------------------------------------------
+
+func (pv *PackageValue) GetShallowSize() int64 {
+	return allocPackage
```

Comment:
I think these should account for all typedvalues as well.

Reply:
That is, the TypedValue array size should be included in the block, even if each value itself might have its own allocation on top.

Reply:
fixed.